ODIN_II/SRC/ast_util.cpp

```diff
@@ -520,7 +521,7 @@ void change_to_number_node(ast_node_t* node, VNumber number) {
 
     node->type = NUMBERS;
     node->types.identifier = temp_ident;
-    node->types.vnumber = new VNumber(number);
+    node->types.vnumber = new VNumber(std::move(number));
```

Comment:
I'd probably keep changes that involve std::move in a separate change to keep it by  `const std::string &`.

(I'd only use std::move if really needed. It more uglifies code otherwise)